fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -597,22 +648,17 @@ public void testCreateVersionOfBinary() throws Exception {
             assertEquals("Binary content of memento must match original content",
                     BINARY_CONTENT, EntityUtils.toString(response.getEntity()));
         }
-    }
 
-    @Test
-    public void testCreateVersionOfBinaryWithDatetime() throws Exception {
-        createVersionedBinary(id);
-
-        final HttpPost createVersionMethod = postObjMethod(id + "/" + FCR_VERSIONS);
-        createVersionMethod.addHeader(MEMENTO_DATETIME_HEADER, MEMENTO_DATETIME);
+        // Verifying that the associated description memento was created
+        final String descriptionMementoUri = mementoUri.replace("fcr:versions", "fcr:metadata/fcr:versions");
 
-        try (final CloseableHttpResponse response = execute(createVersionMethod)) {
-            assertEquals("Must reject memento creation without Content Type",
```

Comment:
This is still a useful test to verify, no? i.e. requirement for a content-type when creating a memento from a provided body.

Reply:
While working on this branch I realized that there didn't appear to be a reason to provide a content type when submitting a historic binary memento. The description is where the content type and other file metadata is provided, which is submitted in a second request, so the file info provided when creating the binary memento would just be ignored (except for checksums which are verified). For that reason I removed the requirement that a content type be provided for binaries.